src/schema/v2/schema.ts

```diff
@@ -225,6 +226,7 @@ export default new GraphQLSchema({
       createGeminiEntryForAsset: CreateGeminiEntryForAsset,
       createUserInterest: createUserInterestMutation,
       deleteCreditCard: deleteCreditCardMutation,
+      deleteUserAccount: deleteUserAccountMutation,
```

Comment:
I'd name this something like `deleteMyAccountMutation`

Reply:
We sorta just followed the example by the mutation directly above this one but maybe that one isn't named real well?

Reply:
Done